activiti-api-impl/activiti-api-task-runtime-impl/src/main/java/org/activiti/runtime/api/impl/TaskAdminRuntimeImpl.java

```diff
@@ -88,6 +90,70 @@ public Task task(String taskId) {
         return new PageImpl<>(tasks,
                               Math.toIntExact(taskQuery.count()));
     }
+    
+    @Override
+    public Task update(UpdateTaskPayload updateTaskPayload) {
+        org.activiti.engine.task.Task internalTask = taskService.createTaskQuery().taskId(updateTaskPayload.getTaskId()).singleResult();
```

Comment:
Lets avoid duplication between `TaskRuntimeImpl` and here. The only difference seems to be that in `TaskRuntimeImpl` we need to check the assignee. You could extract this code inside a new class (`TaskUpdater`) and only check the assignee if the parameter `checkAssignee` is `true`.

Reply:
I have made refactoring. Mock tests for task update are currently **commented**, they should be changed after changes approved...

Reply:
@erdemedeiros 
Can you please check these tests (mock tests, commented) if we need them at all?

Reply:
@CTI777 I'll help you with the test. Please don't comment them out as this might be merged by mistake, it's better to have them failing so we know we need to do something about them.